ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/scenario/OracleMultiUserQueryScenario.scala

```diff
@@ -19,15 +19,15 @@ private[scenario] trait HasRandomCurrency {
   }
 }
 
-object MultiUserQueryScenario {
+object OracleMultiUserQueryScenario {
```

Comment:
renamed since in theory this only makes sense for Oracle right now with the `docker vm` setup